### DISCUSSION TITLE
Virus Total Flagged as Suspicious.Win32.Save.a

### DISCUSSION THREAD

**Discussion by krusem** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#556] [General]
### What went wrong?

The downloaded installer for Windows was flagged by Virus Total as having Suspicious.Win32.Save.a

### Steps to reproduce the bug

Upload installer to Virus Total
https://www.virustotal.com/

### App Version

v1.0.0

### OS Version

Windows 10

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%F0%9F%90%9B+bug%22) and didn't find any duplicate issue
- [X] I have checked the [Frequently Asked Questions (FAQs)](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)) and can confirm that there's no fixes or workarounds for this issue
- [X] This issue only covers one particular bug or issue

**Comment by Melvin-Abraham** [comment]
@krusem Both the installer and the application are safe to use as long as you are downloading from the trusted sources. It probably seems to be a false positive. If you want to confirm any malicious code, you may check the source code of this project. You may also use the `asar` tool to view the packaged contents. If you are still not comfortable, you may build the app yourself (see [How to build](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/blob/master/CONTRIBUTING.md#how-to-build)).

This issue seems similar to #329.

**Reply by thewhitegrizzli** [reply]
> If you want to confirm any malicious code, you may check the source code of this project. (...) you may build the app yourself.
> 
loll..agreed... added to my social boilerplates
